src/components/DisplayNames/DisplayNamesTooltipItem.tsx

```diff
@@ -41,6 +43,7 @@ function DisplayNamesTooltipItem({
     displayName = '',
     textStyles = [],
     childRefs = {current: []},
+    testID = 'displayName',
```

Comment:
```ts
const displayName = 'DisplayNamesTooltipItem';

function DisplayNamesTooltipItem({
    ...
    testID = displayName,
}): DisplayNameTooltipItemProps) {
    ...
}

DisplayNameTooltipItem.displayName = displayName;
```

Reply:
Removed the testID prop as it is no longer needed